app/assets/stylesheets/user.css.scss

```diff
@@ -2,25 +2,27 @@
     padding-top: 150px;
 }
 
+.form-background {
+  display: block;
+  background-color: #eeeeee;
+  padding: 150px 0 20px 20px;
+  text-decoration: none;
+  height: 475px;
+}
+
 #table-title {
-    font-size: 24px;
+    font-size: 32px;
+    font-weight:800;
 }
 
-.register table {
+.form-background table {
     margin-left: auto;
     margin-right: auto;
+    box-shadow: black 1px 1px 1px;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces